packages/message-box/src/index.vue

```diff
@@ -0,0 +1,369 @@
+      type: String,
+      default: '',
+    },
+    modalAppendToBody: {
+      type: Boolean,
+      default: false,
+    },
+    modal: {
+      type: Boolean,
+      default: true,
+    },
+    lockScroll: {
+      type: Boolean,
+      default: true,
+    },
+    showClose: {
+      type: Boolean,
+      default: true,
+    },
+    closeOnClickModal: {
+      type: Boolean,
+      default: true,
+    },
+    closeOnPressEscape: {
+      type: Boolean,
+      default: true,
+    },
+    closeOnHashChange: {
+      type: Boolean,
+      default: true,
+    },
+    center: {
+      default: false,
+      type: Boolean,
+    },
+    roundButton: {
+      default: false,
+      type: Boolean,
+    },
+  } ,
+  setup(props) {
+    let vm
+    const popup = usePopup(props, doClose)
+    const state = reactive<State>({
+      uid: 1,
+      title: undefined,
+      message: '',
+      type: '',
+      iconClass: '',
+      customClass: '',
+      showInput: false,
+      inputValue: null,
+      inputPlaceholder: '',
+      inputType: 'text',
+      inputPattern: null,
+      inputValidator: null,
+      inputErrorMessage: '',
+      showConfirmButton: true,
+      showCancelButton: false,
+      action: '',
+      confirmButtonText: '',
+      cancelButtonText: '',
+      confirmButtonLoading: false,
+      cancelButtonLoading: false,
+      confirmButtonClass: '',
+      confirmButtonDisabled: false,
+      cancelButtonClass: '',
+      editorErrorMessage: null,
+      callback: null,
+      dangerouslyUseHTMLString: false,
+      focusAfterClosed: null,
+      isOnComposition: false,
+      distinguishCancelAndClose: false,
+      type$: '',
+    })
+    const icon = computed(() => state.iconClass || (state.type && TypeMap[state.type] ? `el-icon-${ TypeMap[state.type] }` : ''))
+
+    const confirmButtonClasses = computed(() => `el-button--primary ${ state.confirmButtonClass }`)
+
+    const cancelButtonClasses = computed(() => `${ state.cancelButtonClass }`)
+
+    watch(() => state.inputValue, val => {
+      nextTick().then(() => {
+        if (state.type$ === 'prompt' && val !== null) {
+          validate()
+        }
+      })
+    }, { immediate: true })
+
+    watch(() => popup.state.visible, val => {
+      if (val) {
+        state.uid++
+        if (state.type$ === 'alert' || state.type$ === 'confirm') {
+          nextTick().then(() => { vm.refs.confirm.$el.focus() })
+        }
+        state.focusAfterClosed = document.activeElement
+        dialog = new Dialog(vm.vnode.el, state.focusAfterClosed, getFirstFocus())
+      }
+      if (state.type$ !== 'prompt') return
+      if (val) {
+        setTimeout(() => {
+          if (vm.refs.input && vm.refs.input.$el) {
+            getInputElement().focus()
+          }}, 500)
+      } else {
+        state.editorErrorMessage = ''
+        removeClass(getInputElement(), 'invalid')
+      }
+    })
+
+    onMounted(async () => {
+      vm = getCurrentInstance()
+      await nextTick()
```

Comment:
The code style coule be better. I see there are `nextTick().then`. You should unify the code style